AardvarkLoggingUI.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name     = 'AardvarkLoggingUI'
+  s.version  = '1.0.0'
+  s.license  = 'Apache License, Version 2.0'
+  s.summary  = 'Aardvark is a library that makes it dead simple to create actionable bug reports.'
+  s.homepage = 'https://github.com/square/Aardvark'
+  s.authors  = 'Square'
+  s.source   = { :git => 'https://github.com/square/Aardvark.git', :tag => "AardvarkLoggingUI/#{ s.version.to_s }" }
+
+  s.swift_version = '4.0'
+  s.ios.deployment_target = '8.0'
```

Comment:
again, while we're making a breaking change should we bump the deployment target?

Reply:
Yeah, definitely. I figured I'd leave all of the deployment targets the same for now to avoid any issues with the sample app, and update them all together in a separate PR.